src/Controllers/ReportController.ts

```diff
@@ -154,6 +155,14 @@ module StreamStats.Controllers {
             this.sectionCollapsed = [];
             this.basinCharCollapsed = false;
             this.collapsed = false;
+            this.selectedFDCTMTabName = "";
+            if (this.extensions[0].result.length > 1) {
+                this.extensions[0].result.forEach(r => {
+                    if (r.name.toLowerCase().includes("multivar")) {

```

Comment:
Do you think we should add an `if` statement to limit this "multivar" distinction to just IN and IL for now? It won't make a difference for now, but may reduce unintended behavior for future QPPQ additions.